### DISCUSSION TITLE
Using labels, applications are organized into folders.

### DISCUSSION THREAD

**Discussion by PallaviPalisetti** [gravitational/teleport#14376] [Q&A] [answered]
Hi, there
I'm deploying kube-agent to connect to the root cluster because I have applications hosted on various Kubernetes clusters. Is there a way to use labels or tags to separate the applications into different folders?

thanks

**Comment by webvictim** [comment, marked as answer]
Yes, you can set key/value pairs of labels for each `teleport-kube-agent` using the `labels:` value: https://github.com/gravitational/teleport/blob/master/examples/chart/teleport-kube-agent/values.yaml#L78-L79

```yaml
labels:
  env: test
  cloud: aws
  region: us-east-1
```

**Reply by PallaviPalisetti** [reply]
is there a way to separate applications into folders based on labels

**Reply by webvictim** [reply]
No. You can filter by labels in the UI or via `tsh ls --query` which can provide a similar experience: https://goteleport.com/docs/setup/reference/predicate-language/

**Reply by PallaviPalisetti** [reply]
thanks for quick response. how to remove the default labels for applications example: **teleport.dev/origin: config-file**

**Reply by webvictim** [reply]
Those are always displayed and can't currently be hidden.
